src/battle/common/actor/slot_machine.inc.c

```diff
@@ -121,7 +121,7 @@ API_CALLABLE(N(Add1Coin)) {
 EvtScript N(init) = {
 #if VERSION_PAL
     EVT_CALL(GetLanguage, LVar0)
-    EVT_IF_EQ(LVar0, 3)
+    EVT_IF_EQ(LVar0, LANGUAGE_ES)
         EVT_SET(LVar0, 2)
```

Comment:
This is still an enum value, so LANGUAGE_FR here. Likewise elsewhere this pattern occurs in the scripts.